scripts/assemble_changelog.py

```diff
@@ -448,14 +453,23 @@ def list_files_to_merge(options):
     files_to_merge.sort(key=EntryFileSortKey)
     return files_to_merge
 
+def check_extensions(options):
+    files = glob.glob(os.path.join(options.dir, '*'))
+    files = {x for x in files if not x.endswith(".txt")}
+    files.discard("ChangeLog.d/00README.md")
```

Comment:
Skipping `README.md` this way doesn't work with a different `options.dir` or on Windows.
```suggestion
    files = {x for x in files if not x.endswith(".txt") and not os.path.basename(x) == '00README.md'}
```